guides/gatherers.md

```diff
@@ -14,32 +14,49 @@ Facts Gathering process in a nutshell
 fact = gatherer(argument)
 ```
 
+## Gatherers versioning
+
+When an update to the trento-agent includes a non-backwards compatible change in a gatherer (e.g., changes to the Rhai output format), its version is
```

Comment:
I think we should have a better 1st phrase. Something like:
```
The gatherers implementation supports a versioning mechanism in order to enable non-backwards compatibility changes in any of them.
When an update to the trento-agent includes a non-backwards...
```
At the end, we want to start describing a bit the chapter, not just how it is implemented